pkg/utils/utils.go

```diff
@@ -32,9 +32,10 @@ const (
 
 // COMPONENT_IMAGES are thee keys are based on the "container name" + "-{image,version}"
 var COMPONENT_IMAGES = map[string]string{
-	"curator":             "CURATOR_IMAGE",
-	constants.FluentdName: constants.FluentdImageEnvVar,
-	"kibana":              "KIBANA_IMAGE",
+	"curator":               "CURATOR_IMAGE",
+	constants.FluentdName:   constants.FluentdImageEnvVar,
+	constants.CollectorName: constants.FluentdImageEnvVar,
```

Comment:
Why do we have two variables with same value ? Is it because all naming transformation is WIP ?